tests/integration_tests/Test/PM/Integration/FileBinaryTar.cls

```diff
@@ -20,7 +20,7 @@ Method TestPackageAndExtract()
     Set outFile = ##class(%Library.File).TempFilename()
     Set outDir = ##class(%Library.File).NormalizeDirectory(##class(%Library.File).TempFilename()_"dir-out")
     Do ##class(%Library.File).CreateDirectoryChain(outDir)
-    Do $$$AssertEquals($zf(-100,"/STDOUT="_outFile_"/STDERR="_outFile,"tar","-xvf",tempDir_".tgz","-C",outDir),0)
+    Do $$$AssertEquals($zf(-100,"/STDOUT="""_outFile_"""/STDERR="""_outFile_"""","tar","-xvf",tempDir_".tgz","-C",outDir),0)
```

Comment:
Not related to this PR per se, but docs.intersystems.com is vague (if not wrong) on usage of $zf(-100)

> If /STDOUT=filename and /STDERR=filename (or /STDOUT+=filename and /STDERR+=filename) specify the same filename, the specified file is only opened or created once. The resulting file handle is duplicated and supplied as both the stdout and stderr file handles given to the process that executes the specified cmd string.

Reply:
Yeah I already provided "Feedback" on this (internal ref: DOCS-14241) and wrote up https://community.intersystems.com/post/debugging-error-zf-100-only-happens-linux